test/Microsoft.AspNet.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -155,7 +155,7 @@ public class FiltersTest
                 "http://localhost/AuthorizeUser/Impossible");
 
             // Assert
-            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
```

Comment:
Ok, cool. I'll sign off on this change, but can you send a PR to add the missing test with `[Fact(Skip = "blah blah")]`. I don't think anyone else is aware of the bug.